src/components/Combobox/Static/StaticCombobox.stories.tsx

```diff
@@ -1,15 +1,17 @@
 import { Meta, StoryObj } from "@storybook/react";
 import { useState } from "react";
 
+import { Box } from "~/components/Box";
+import { Option } from "~/components/BaseSelect";
 import { Combobox } from "..";
 
 const options = [
-  { value: "color-black", label: "Black" },
-  { value: "color-red", label: "Red" },
-  { value: "color-green", label: "Green" },
-  { value: "color-blue", label: "Blue" },
-  { value: "color-orange", label: "Orange" },
-  { value: "color-purple", label: "Purple" },
+  { value: "black", label: "Black" },
```

Comment:
**Question:** why changing `color-black` to `black`?

It was added to add more difference between value and label - as in real world scenario those values will be some kind of ids etc